### DISCUSSION TITLE
where to buy the Create3 battery

### DISCUSSION THREAD

**Discussion by nidia-bucarelli** [iRobotEducation/create3_docs#265] [Questions] [answered]
Hi,
I got the Turtlebot4 which uses the iRobot Create 3 as a base. The battery is damaged, and I would like to replace it.

This is the battery:
![imagen](https://user-images.githubusercontent.com/112603498/205214628-9ee97121-8381-4e7f-9685-1d1a663979c1.png)

I cannot find the exact battery. I was able to find the ones below. But they have  some different serial/specs, which I dont understand. Will any of these batteries do the work? If not, where can I purchase the Create3 battery?
 
![imagen](https://user-images.githubusercontent.com/112603498/205214845-789fedca-19ec-4f07-a3bd-db788b55958c.png)

![imagen](https://user-images.githubusercontent.com/112603498/205215338-b8cb94ee-c812-4d10-b3eb-d43eef2e6a79.png)

**Comment by shamlian** [comment, marked as answer]
Create 3 is designed upon the Roomba i3 platform, and consequently uses the [Lithium Ion Battery for Roomba® e & i series](https://www.irobot.com/en_US/roomba-accessories/4624864.html). These can be purchased from most vendors which sell iRobot products.

However, I am concerned to hear that you're saying the battery is damaged; it's my personal opinion that you should pursue a warranty replacement with Clearpath.
